pyanaconda/ui/gui/spokes/software_selection.py

```diff
@@ -526,6 +556,22 @@ def _get_selected_addons(self):
 
         return retval
 
+    def _get_kernel_from_properties(self):
```

Comment:
With a little effort, this method could be replaced with a function that could be tested with unit tests. We usually use `pyanaconda.ui.lib` for this kind of code.